resources/js/scriptchart.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function () {
+    var ctx = document.getElementById('barChart');
+    const data = ctx.getAttribute('chart-data');
+    var myChart = new Chart(ctx, {
+        type: 'bar',
+        data: {
+            datasets: [{
+                label: 'Revenue statistics',
+                data: JSON.parse(data),
+                backgroundColor: [
+                    'rgba(255, 99, 132, 0.2)',
+                    'rgba(255, 159, 64, 0.2)',
+                    'rgba(255, 205, 86, 0.2)',
+                    'rgba(75, 192, 192, 0.2)',
+                    'rgba(54, 162, 235, 0.2)',
+                    'rgba(153, 102, 255, 0.2)',
+                    'rgba(201, 203, 207, 0.2)',
+                    'rgba(255, 205, 86, 0.2)',
+                    'rgba(75, 192, 192, 0.2)',
+                    'rgba(54, 162, 235, 0.2)',
+                    'rgba(153, 102, 255, 0.2)',
+                    'rgba(201, 203, 207, 0.2)'
```

Comment:
```suggestion
                    'rgba(201, 203, 207, 0.2)',
```
Fix tương tự các chỗ khác

Reply:
chưa fix